internal/venice-client-common/src/main/java/com/linkedin/venice/schema/writecompute/WriteComputeSchemaConverter.java

```diff
@@ -318,91 +319,24 @@ private Schema convertMap(Schema mapSchema, String name, String namespace) {
   }
 
   /**
-   * Wrap up a union schema with possible write compute operations.
-   *
-   * N.B.: if it's an top level union field, the convert will try to convert the elements(except for RECORD)
-   * inside the union. It's not supported if an union contains more than 1 collections since it will
-   * confuse the interpreter about which elements the operation is supposed to be applied.
-   *
-   * e.g.
-   * original schema:
-   *{
-   *   "type" : "record",
-   *   "name" : "record",
-   *   "fields" : [ {
-   *     "name" : "nullableArrayField",
-   *     "type" : [ "null", {
-   *       "type" : "array",
-   *       "items" : {
-   *         "type" : "record",
-   *         "name" : "simpleRecord",
-   *         "fields" : [ {
-   *           "name" : "intField",
-   *           "type" : "int",
-   *           "default" : 0
-   *         } ]
-   *       }
-   *     } ],
-   *     "default" : null
-   *   } ]
-   * }
-   *
-   * write compute schema:
-   * {
-   *   "type" : "record",
-   *   "name" : "testRecordWriteOpRecord",
-   *   "fields" : [ {
-   *     "name" : "nullableArrayField",
-   *     "type" : [ {
-   *       "type" : "record",
-   *       "name" : "NoOp",
-   *       "fields" : [ ]
-   *     }, "null", {
-   *       "type" : "record",
-   *       "name" : "nullableArrayFieldListOps",
-   *       "fields" : [ {
-   *         "name" : "setUnion",
-   *         "type" : {
-   *           "type" : "array",
-   *           "items" : {
-   *             "type" : "record",
-   *             "name" : "simpleRecord",
-   *             "fields" : [ {
-   *               "name" : "intField",
-   *               "type" : "int",
-   *               "default" : 0
-   *             } ]
-   *           }
-   *         },
-   *         "default" : [ ]
-   *       }, {
-   *         "name" : "setDiff",
-   *         "type" : {
-   *           "type" : "array",
-   *           "items" : "simpleRecord"
-   *         },
-   *         "default" : [ ]
-   *       } ]
-   *     }, {
-   *       "type" : "array",
-   *       "items" : "simpleRecord"
-   *     } ],
-   *     "default" : { }
-   *   } ]
-   * }
+   * Wrap up a union schema with possible partial update operations.
+   * Only two branches union with one being NULL type is allowed. This will make partial update logic compatible with
+   * Active/Active replication schema requirement.
```

Comment:
Is it possible to add what's the Active/Active replication schema requirement with a sentence or two? It'd be easier for people later to understand this method.